File: Code/App/v2/Patient/Scripts/App/Model/PersonalData/PatientsDieseasesModel.js
function PatientsDieseasesModel(data) {
    var self = this;
        
    this.Id = ko.observable(data.Id);
    this.DieseaseName = ko.observable(data.DieseaseName);
    this.Description = ko.observable(data.Description);
        
    this.DieseasesExaminations = ko.observableArray([]);

    if (data.DieseasesExaminations != null) {
        data.DieseasesExaminations.forEach(function (item) {
            self.DieseasesExaminations.push(new ExaminationModel(item));
        });
    };

};

File: Code/App/separateDB/RepositoryClasses/Models/PatientsDieseasesModel.cs
using System.Collections.Generic;

namespace RepositoryClasses.Models
{
    public class PatientsDieseasesModel
    {
        public int Id { get; set; }
        public string DieseaseName { get; set; }
        public string Description { get; set; }
        public IEnumerable<ExaminationsModel> DieseasesExaminations { get; set; }
        public int ExtPatientDieseaseId { get; set; }
    }
}